listenbrainz/webserver/static/js/src/UserPageHeading.tsx

```diff
@@ -24,29 +24,37 @@ import * as Sentry from "@sentry/react";
 import FollowButton from "./FollowButton";
 import APIService from "./APIService";
 import GlobalAppContext, { GlobalAppContextT } from "./GlobalAppContext";
+import ReportUserButton from "./ReportUser";
 import { getPageProps } from "./utils";
 
 const UserPageHeading = ({
   user,
   loggedInUser,
   loggedInUserFollowsUser = false,
+  alreadyReportedUser = false,
 }: {
   user: ListenBrainzUser;
   loggedInUser: ListenBrainzUser | null;
   loggedInUserFollowsUser: boolean;
+  alreadyReportedUser: boolean;
 }) => {
   return (
-    <h2 className="page-title">
-      {user.name}
-      {loggedInUser && user.name !== loggedInUser.name && (
-        <FollowButton
-          type="icon-only"
-          user={user}
-          loggedInUser={loggedInUser}
-          loggedInUserFollowsUser={loggedInUserFollowsUser}
-        />
+    <>
+      <h2 className="page-title">
+        {user.name}
+        {loggedInUser && user.name !== loggedInUser.name && (
+          <FollowButton
+            type="icon-only"
+            user={user}
+            loggedInUser={loggedInUser}
+            loggedInUserFollowsUser={loggedInUserFollowsUser}
+          />
+        )}
+      </h2>
+      {loggedInUser && (
```

Comment:
We should probably have the `user.name !== loggedInUser.name` here as well.

Reply:
I don't think that's an issue, since the API does that check.
I'd rather the button be visible in the same place at all times so that users know where to find it (it's likely that the most time they will spend is on their own page; seeing the button there cements it in place)

Reply:
Makes sense. :+1: